lib/compress/zstd_compress.c

```diff
@@ -659,6 +659,24 @@ ZSTD_compressionParameters ZSTD_adjustCParams(ZSTD_compressionParameters cPar, u
     return ZSTD_adjustCParams_internal(cPar, srcSize, dictSize);
 }
 
+static size_t ZSTD_sizeof_matchState(ZSTD_compressionParameters const* cParams, const U32 cctx)
```

Comment:
minor detail :
`cctx` is generally a variable name for a context pointer,
but here it's more like a flag.
Could this name be changed to reduce confusion ?